core/examples/json/queryables.json

```diff
@@ -5,68 +5,68 @@
             "type": "string"
         },
         {
-            "queryable": "externalId",
-            "type": "string"
+            "queryable": "created",
+            "type": "datetime"
         },
         {
-            "queryable": "title",
-            "type": "string"
+            "queryable": "updated",
+            "type": "datetime"
         },
         {
-            "queryable": "description",
+            "queryable": "conformsTo",
             "type": "string"
         },
         {
-            "queryable": "keywords",
+            "queryable": "title",
             "type": "string"
         },
         {
             "queryable": "type",
             "type": "string"
         },
         {
-            "queryable": "publisher",
-            "type": "datetime"
+            "queryable": "description",
+            "type": "string"
         },
         {
-            "queryable": "themes",
-            "type": "datetime"
+            "queryable": "keywords",
+            "type": "string"
```

Comment:
How's the type determined? Isn't that actually an array of strings? Maybe I'm missing something though...

Reply:
I've updated the type based on the type of the actual queryable.  For example, if `keywords` is an array, the queryable is of type `array`.  Users searching would pass a string literal (`keywords=birds`).  @pvretano is this the right interpretation of defining queryables?

Reply:
That makes sense, I've just seen other implementations (in STAC?), where the queryables are 1:1 mappings to the actual "property" (here array). This left me confused. If this PR is the correct way forward, great.